ROADMAP.md

```diff
@@ -0,0 +1,11 @@
+# GUAC Roadmap
+
+At a high level, the roadmap for GUAC will include
+
+- Persistent Database
+- Associate Identity and Trust with various GUAC artifacts
+- VEX support
+- Support for license related information
+- ITE6 support via predicate dictionary
+- Evidence store for SBOMs and attestations
+- Expanded integration via other "CHIPS"
```

Comment:
nit: i'd say to just be a bit more descriptive here (add CLI utilities like existing x, y, z) since we don't quite define CHIPS yet

Reply:
Updated.